src/core/wordle/db.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I think that the content of this file should be added to `utils/database/db.json` file that already exists. this will make sure that there is a single database file.

Reply:
I wanted to discuss this also, well in `core/advises` there is a kind of database for advice, and the database of dadjokes were put separately in `utils`, as the datasources for each of the existing commands were separated, I separated this as well. But I think  (and might not be right) `core` is more suitable for data then `utils`.

Reply:
when I added `dadjokes`, there was no management for a advice database. so it was created in the utils folder. but you can move that folder to core and make it a central place for all database interactions, and maybe in the future you refactor advice to use the same approach. let see what @youben11 thinks.

Reply:
I'm not sure either, these for me are db files and doesn't have to follow a specific structure as if it were a module for db management. IMO they can be wherever they are used, so `core/wordle` is fine with me